### PR TITLE
fix: make constants reader strict

### PR DESCRIPTION
**Pull request by guanbinrui**
closes #

### CODE REVIEW

**Review by UncleBill — APPROVED** [review]

File: packages/maskbook/src/plugins/Airdrop/hooks/useAirdropPacket.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useAsyncRetry } from 'react-use'
 import { AirdropRPC } from '../messages'
 
-export function useAirdropPacket(address: string) {
+export function useAirdropPacket(address?: string) {
```
Comment (septs) [inline packages/maskbook/src/plugins/Airdrop/hooks/useAirdropPacket.ts:4]:
```suggestion
export function useAirdropPacket(address: string | undefined) {
```

**Comment by nuanyang233** [comment]
plz fix eslint warning

**Comment by zhouhanseng** [comment]
After we implement `postInspectorRequiredChains`, we can ensure that some plugin-related constants are not undefined.

**Review by zhouhanseng — DISMISSED** [review]

**Comment by guanbinrui** [comment]
> After we implement `postInspectorRequiredChains`, we can ensure that some plugin-related constants are not undefined.

Perfect! But the constants reader could be undefined. The type should be fixed.

**Pull request merged**